client/src/main/scala/com/dwolla/cloudflare/ZoneClient.scala

```diff
@@ -9,6 +9,7 @@ import org.http4s._
 
 trait ZoneClient[F[_]] {
   def getZoneId(domain: String): Stream[F, ZoneId]
+  def getAccountId(domain: String): Stream[F, AccountId]
```

Comment:
This addition will also be used in `cloudflare-lambda`. It allows a user defining an `AccessControlRule` to still only need to provide a zone. `cloudflare-lamdba` can then extrapolate the account id given this information.

Reply:
It's not specific per se. I can go ahead and remove the need for this method by exposing additional methods on `AccessControlRuleClient` that take in a zone id.  For example:

```
def getById(zoneId: Zone, ruleId: String): Stream[F, AccessControlRule] = ...
```

The body of the method will utilize Cloudflare's API endpoint specific for zone-level rules.

Reply:
This should now be addressed. Note that I tweaked my initial suggested method signature slightly to:

```
override def getById(level: Level, ruleId: String): Stream[F, AccessControlRule] = ...
```

where `Level` is defined as either an account or zone:

```
sealed trait Level
object Level {
  case class Account(accountId: AccountId) extends Level
  case class Zone(zoneId: ZoneId) extends Level
}
```